docs/concepts/07_dip/02_provider.md

```diff
@@ -0,0 +1,81 @@
+---
+id: provider-pallet
+title: Provider pallet
+---
+
+The Decentralized Identity Provider (DIP) provider pallet enables a Substrate-based chain (provider) to bridge the identities of its users to other connected chains (consumers) trustlessly.
+A consumer chain is connected to a provider if there is a way for the consumer chain to verify state proofs about parts of the state of the provider chain.
+
+The pallet is agnostic over the chain-specific definition of identity and delegates the definition of it to the provider chain's runtime.
+
+Because providers and consumers evolve at different speeds, identity commitments are versioned.
+This allows the provider chain to upgrade to a newer commitment scheme while allowing users to use the old version if the chains on which they want to use their identity don't yet support the new scheme.
+
+The identity subject can replace or remove identity commitments, for example, if something in the identity info changes.
+After removal, the identity becomes unusable cross-chain, although it continues to exist on the provider chain and is usable for local operations.
+
+#### Add the pallet to the runtime
+
+Add the pallet to runtime in your `Cargo.toml` file dependencies section:
+
+:::info
+
+The DIP pallet is available in the KILT node release 1.12.0 and later.
+
+:::
+
+```toml
+provider = {package = "pallet-dip-provider", git = "https://github.com/KILTprotocol/kilt-node.git", branch = "release-1.12.0"}
+```
+
+#### Configure the pallet
+
+Pass runtime configuration to the pallet's `Config` trait.
+
+```rust
+impl pallet_dip_provider::Config for Runtime {
+    // Config
+}
+```
+
+<!-- TODO: Link for full details -->
+
+The following configuration parameters are required. Read the xxx :
```

Comment:
I presume this is a place holder :)

Reply:
Do you want to merge with the placeholder in? If not, please regard this comment as a reminder about the placeholder before merging 😄